### PR TITLE
Add arguments to build.sh and destroy.sh

### PR DESCRIPTION
**Pull request by mpagot**
build and destroy glue sripts support new arguments:
-k to provide th essh key path
-q to tune the output for openQA
-s to skip the Ansible part
Optimize the Dockerfile by removing a layer and to pipe curl to tar
Glue script to POSIX (an dno more bash)
Change the readme to document the new arguments
Add awscli to the requirements.txt file

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # syntax=docker/dockerfile:experimental
 
 FROM opensuse/tumbleweed:latest
```
Comment (mr-stringer) [inline Dockerfile:3]: Should we use latest or stick to a known release?  If something changes and breaks our automation, we'd most likely blame the automation.

File: Dockerfile [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-RUN zypper ref && zypper up -y
```
Comment (mr-stringer) [inline Dockerfile:4]: Same here, it won't give us a stable environment.

File: Dockerfile (hunk 1 above)
Comment (alvarocarvajald) [reply Dockerfile:3]: Known release has my vote.

**Comment by alvarocarvajald** [comment]
I am not sure whether `terraform/azure/terraform.tfvars.openqa` should be in this repo or in os-autoinst-distri-opensuse. IMO it makes more sense to centralize openQA related assets in the openQA repositories.

**Comment by mpagot** [comment]
> I am not sure whether `terraform/azure/terraform.tfvars.openqa` should be in this repo or in os-autoinst-distri-opensuse. IMO it makes more sense to centralize openQA related assets in the openQA repositories.

Content of this file (list of variables like **%DEPLOYMENTNAME%** that openQA will replace with proper values) has to be known by openQA test code (or library). 

On the other hand, the content of this .tfvars.openqa has to be aligned to any new variables introduced by the qe-sap-deployment, as anyone is touching the .tfvars.example is supposed to touch this one too (that otherwise is sort of hidden in another repo)

**Comment by alvarocarvajald** [comment]
> On the other hand, the content of this .tfvars.openqa has to be aligned to any new variables introduced by the qe-sap-deployment, as anyone is touching the .tfvars.example is supposed to touch this one too (that otherwise is sort of hidden in another repo)

Yes, that's a valid counterpoint for having it here.

**Pull request merged**